addon/components/docs-demo/styles.scss

```diff
@@ -0,0 +1,5 @@
+.docs-demo__snippets-nav-item {
```

Comment:
We have an `outline-none` class defined [here](https://github.com/ember-learn/ember-cli-addon-docs/blob/master/addon/tailwind/config/modules.css#L142) that we can use instead of BEM

I know discovery is a problem, I have some ideas for how to fix it

Reply:
Ah okay. I'll clean that up.

Yeah, diving into the CSS has been a little hard. I have no idea which (of the non-BEM classes) are part of this app's styles and which are part of Tailwind. There are a ton of BEM classes in the templates that aren't actually defined in the CSS anywhere, and a bunch of `classNames` in JS that are just commented out 🤷‍♂️